clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -353,13 +379,22 @@ void testSendUnsentRequest() {
         assertTrue(networkClient.hasAnyPendingRequests());
         assertFalse(networkClient.unsentRequests().isEmpty());
         assertFalse(client.hasInFlightRequests());
-        consumerNetworkThread.cleanup();
+        consumerNetworkThread1.cleanup();
```

Comment:
nit: up to you but we really don't need this change, and could keep the var name just as `consumerNetworkThread`